### PR TITLE
fix build issue on Alpinelinux

### PR DESCRIPTION
**Pull request by zaoqi**
this PR depens on https://github.com/AppImage/libappimage/pull/145

### CODE REVIEW

**Comment by probonopd** [comment]
Do you see any issues with this @TheAssassin?

File: src/light_elf.h [hunk 1]
```diff
@@ -26,7 +26,9 @@
 
 #include <inttypes.h>
 
-__BEGIN_DECLS
+#ifdef _cplusplus
```
Comment (TheAssassin) [inline src/light_elf.h]: Doesn't it have to be `__cplusplus`? See e.g., https://renenyffenegger.ch/notes/development/languages/C-C-plus-plus/preprocessor/macros/predefined/__cplusplus.

**Comment by TheAssassin** [comment]
@probonopd the change itself is harmless, it won't interfere with our build system (if that macro doesn't exist, the `extern C` will just be ignored). However it doesn't quite seem right. @zaoqi please double-check your commit.

File: src/light_elf.h (hunk 1 above)
Comment (zaoqi) [reply src/light_elf.h]: fixed

**Comment by zaoqi** [comment]
ping @TheAssassin

**Comment by probonopd** [comment]
Thank you very much gentlemen :+1:

**Pull request merged**